feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/SyncInfoViewModel.kt

```diff
@@ -188,20 +189,29 @@ internal class SyncInfoViewModel @Inject constructor(
         isSyncRunning: Boolean?,
         isConnected: Boolean?,
         syncConfiguration: SynchronizationConfiguration? = configuration.value?.synchronization,
-    ) = isConnected == true &&
-        isSyncRunning == false &&
-        syncConfiguration?.let {
-            !isModuleSync(it.down) ||
-                isModuleSyncAndModuleIdOptionsNotEmpty(
-                    it,
-                )
-        } == true
+    ): Boolean {
+        if (isSyncRunning == true) return false
+
+        val simprintsDownConfig = syncConfiguration?.down?.simprints
+        if (simprintsDownConfig != null &&
+            isConnected == true &&
+            (!isModuleSync(simprintsDownConfig) ||
+             isModuleSyncAndModuleIdOptionsNotEmpty(simprintsDownConfig))
+        ) {
+            return true
+        }
+
+        val commCareDownConfig = syncConfiguration?.down?.commCare
+        if (commCareDownConfig != null) return true
+
+        return false
```

Comment:
The last couple of lines are a bit redundant and could be just 
`return syncConfiguration?.down?.commCare != null`

Reply:
As I said - IMO it conveys the meaning much better and is therefore easier to understand.

Reply:
But the meaning is the same, just conveyed in a more concise form.


This version is basically writing
```
fun isEven(n: Int): Boolean {
  if (n % 2 == 0) {
    return true
  }
  return false
}
```
instead of 
```
fun isEven(n: Int) = n % 2 == 0
```